### ISSUE TITLE
cerendin halgar not giving carendin summons,devourer of souls,hearts of the pure,the binding 

### ISSUE THREAD

**Issue by Antiotpad** [dalaranwow/dalaran-wow#4899]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Antiotpad** [comment]
![wowscrnshot_101317_123017](https://user-images.githubusercontent.com/32479624/31542719-7e8814ce-b013-11e7-992c-89f7d8b65ceb.jpg)

**Comment by Antiotpad** [comment]
![wowscrnshot_101317_124113](https://user-images.githubusercontent.com/32479624/31542818-d4246914-b013-11e7-9b02-a1a9fcbb6338.jpg)

**Comment by TeridanDW** [comment]
I am having really hard time understanding what you are trying to report here @Antiotpad , could you fill the  templete (Current behaviour, Expected behaviour, Steps to reproduce the problem, Include proofs for this behaviour) and I will check it on PTR.

**Comment by Antiotpad** [comment]
no npc's giving succubus quests.. its bugged and i cant fix that

**Comment by Antiotpad** [comment]
![wowscrnshot_101317_130329](https://user-images.githubusercontent.com/32479624/31543763-f820ee42-b017-11e7-917e-32fe1eae5134.jpg)

**Comment by Antiotpad** [comment]
![wowscrnshot_101317_131502](https://user-images.githubusercontent.com/32479624/31543884-8b0893d6-b018-11e7-9db8-bae0951ff40e.jpg)

**Comment by Antiotpad** [comment]
enough of proofs????

**Comment by Antiotpad** [comment]
current behaviour : npc cerendin halgar won't give quest carendin summons 
expected behaviour : cerendin halgar should give quest carendin summons

**Comment by TeridanDW** [comment]
> enough of proofs????

Your screenshots does not proof that the quest/mechanic or something else serverside is not working as intended, just that you don't see the quest you think you should see.
After some research I found that quest chain you are looking for. For horde players there are 2 chains:
 - Quest https://wowgaming.altervista.org/aowow/?quest=1507 in Orgrimmar
 - Quest https://wowgaming.altervista.org/aowow/?quest=10605 (The one that you are looking for) **that starts in Silvermoon City and then leads to Undercity.**

Both of these chains ends with the player getting spell https://wowgaming.altervista.org/aowow/?spell=712 and item https://wowgaming.altervista.org/aowow/?item=22243

Note: Posting personal(character) issues on bug tracker will **NOT** be prioritised by the GMs in any way.

Closing as there is nothing serverside to be fixed here.

**Comment by Antiotpad** [comment]
![wowscrnshot_101317_144346](https://user-images.githubusercontent.com/32479624/31547430-a12f4d44-b027-11e7-8e56-5e729342e95d.jpg)

**Comment by Antiotpad** [comment]
![wowscrnshot_101317_145926](https://user-images.githubusercontent.com/32479624/31547464-b94b0e04-b027-11e7-92c2-7dd4f3ae27f3.jpg)

**Comment by Antiotpad** [comment]
![wowscrnshot_101317_145954](https://user-images.githubusercontent.com/32479624/31547477-c5ddde1c-b027-11e7-819c-6dc92645a6f0.jpg)

**Comment by Antiotpad** [comment]
![wowscrnshot_101317_150027](https://user-images.githubusercontent.com/32479624/31547485-ce8723d4-b027-11e7-9f24-8ff57d0ce545.jpg)

**Comment by Antiotpad** [comment]
can you tell me where to search npc?? i dont belive there is any npc that start that quest chain

**Comment by Antiotpad** [comment]
![wowscrnshot_101317_150908](https://user-images.githubusercontent.com/32479624/31547658-7e4f48b4-b028-11e7-9f3a-bb03ab3107dc.jpg)

**Comment by Antiotpad** [comment]
so how i exactly prove that there is no quest?
